python/llm/src/bigdl/llm/transformers/modelling_bigdl.py

```diff
@@ -20,58 +20,209 @@
 # only search the first bigdl package and end up finding only one sub-package.
 
 from bigdl.llm.utils.common import invalidInputError
+from .model import AutoModelForCausalLM
 
 
-class BigdlNativeForCausalLM:
-    """
-    A generic model class that mimics the behavior of
-    ``transformers.LlamaForCausalLM.from_pretrained`` API
-    """
+class LlamaForCausalLM:
 
     @classmethod
     def from_pretrained(cls,
                         pretrained_model_name_or_path: str,
-                        model_family: str = 'llama',
-                        dtype: str = 'int4',
+                        native: bool = True,
+                        dtype: str = "int4",
```

Comment:
This is only valid when native=True? If so, do we need to add something in the doc?

Reply:
`native = True` -> Native int4; 
`native = False` -> Transformers int4